### ISSUE TITLE
Titanium

### ISSUE THREAD

**Issue by Dr-Mic** [FantasyTeddy/QuantumFlux#13]
Will titanium make a return in this updated mod? If I remember correctly, it was used to make the armor and a weapon.

**Comment by FantasyTeddy** [comment]
Yes I think titanium will eventually return as soon as I integrate the battlesuit armor again (see issue #9).
